assets/js/src/newsletters/send.jsx

```diff
@@ -81,13 +78,12 @@ class NewsletterSend extends React.Component {
 
   isValid = () => jQuery('#mailpoet_newsletter').parsley().isValid();
 
-  isValidFromAddress = () => {
+  isValidFromAddress = async () => {
```

Comment:
Could you please keep a condition for MailPoet sending method here?
```
if (window.mailpoet_mta_method !== 'MailPoet') {	    
      return true;	
}
```
It is handled in `loadAuthorizedEmailAddresses`  but current code in function `isValidFromAddress ` was a bit confusing for me because a condition `if (addresses === false) return true;` looks like we mark from address as valid when user has no authorized addresses.